protostar/commands/test/test_execution_environment.py

```diff
@@ -180,7 +187,10 @@ def build_cheatcodes(
             internal_calls: List[CallInfo],
         ) -> List[Cheatcode]:
             data_transformer = DataTransformerFacade(self._starknet_compiler)
-            declare_cheatcode = DeclareCheatcode(syscall_dependencies)
+            declare_cheatcode = DeclareCheatcode(
+                syscall_dependencies,
+                disable_hint_validation=self._disable_hint_validation_in_external_contracts,
```

Comment:
Wouldn't it be better to put it inside starknet_general_config? Would be consistent with `cairo_path` as this are both compiler args

Reply:
List of such populated arguments is very likely to grow in the future

Reply:
As discussed, moved cairo_path as a dependency of `DeclareCheatcode`. Removed `CheatableStarknetGeneralConfig` because it was no longer used.